validator.js

```diff
@@ -10,6 +10,7 @@ const defaultMessages = {
   'label.error.phoneNumber': 'Sorry, please enter a valid phone number.',
   'label.error.range': 'Sorry, please enter a valid number.',
   'label.buttonHint.default': "Hello, we just wanted to send a friendly follow up. If you would like to stop the survey, just ignore this message and we won't bother you again.",
+  'label.off': 'Sorry, the survey is closed.',
```

Comment:
So the other ones are "label.something" because they come from Typeform. Given that this doesn't, I find it confusing to use the same convention. How is it going to be filled, exactly?

Reply:
I can see those labels being pulled from the DB once we start working on the ticket about exposing them to the `dashboard-client`. 

For consistency to the other labels I labeled `label.off.`, but this _entire_ dict may be gone in the future.

Reply:
Ok, sounds good!